packages/common/src/store/pages/token-dashboard/slice.ts

```diff
@@ -293,6 +293,12 @@ const slice = createSlice({
     },
     resetRemovedStatus: (state) => {
       state.associatedWallets.removeWallet.status = null
+    },
+    resetState: (state) => {
+      state.associatedWallets = initialState.associatedWallets
```

Comment:
Is there are reason for only resetting parts of this slice? Why not set back to the initialState

Reply:
for some weird reason doing `state = initialState` doesn't seem to work...

Reply:
I've seen this weirdness with immer before, can you try returning initial state from this reducer?

Reply:
great idea, will give that a go